storage/innobase/include/trx0trx.ic

```diff
@@ -357,12 +357,14 @@ trx_is_high_priority(const trx_t* trx)
 @return the transaction that will be rolled back, null don't care */
 UNIV_INLINE
 const trx_t*
-trx_arbitrate(const trx_t*	requestor,
-	      const trx_t*	holder
 #ifdef WITH_WSREP
-	      ,my_bool		sync
-#endif /* WITH_WSREP */
+trx_arbitrate(const trx_t*	requestor,
```

Comment:
In the header file we have ifndef for this function declaration. It is consistent now, but makes a confusion.